packages/auth/src/AuthCard/AuthCard.tsx

```diff
@@ -33,38 +33,30 @@ export const AuthCard = ({
   const { logo } = React.useContext(LogoContext);
 
   return (
-    <Card variant="forms.auth.cardContainer">
+    <Card variant="primary" sx={{ maxWidth: '340px' }}>
       <Flex sx={{ flexDirection: 'column', gap: 3 }}>
         {logo && (
           <Flex sx={{ width: '100%', justifyContent: 'center' }}>{logo}</Flex>
         )}
 
         {children}
 
-        <Flex variant="forms.auth.submitButtonContainer">
-          <Button
-            type="submit"
-            aria-label="submit-login"
-            variant="forms.auth.submitButton"
-          >
+        <Flex sx={{ justifyContent: 'space-between', marginTop: 7 }}>
+          <Button type="submit" aria-label="submit-login" variant="primary">
             {buttonLabel}
           </Button>
           {forgotPassword && (
-            <Link variant="forms.auth.link" onClick={forgotPassword.onClick}>
+            <Link variant="primary" onClick={forgotPassword.onClick}>
```

Comment:
I think we don't need the variant prop here. We can assume links will be the default always. If not, we use a variant to assign the distinctive link style.

The same idea is valid to all links of this component.